.travis.yml

```diff
@@ -2,31 +2,25 @@ os: linux
 
 dist: xenial
 
-language: c
+language: python
```

Comment:
looks strange... explaining comment would be good

Reply:
done